python/mspasspy/db/database.py

```diff
@@ -3804,8 +4421,112 @@ def index_mseed_file(self, dfile, dir=None, collection='wf_miniseed'):
         ind = _mseed_file_indexer(fname)
         for i in ind:
             doc = self._convert_mseed_index(i)
-            doc['storage_mode'] = 'file'
-            doc['format'] = 'mseed'
-            doc['dir'] = odir
-            doc['dfile'] = dfile
+            doc["storage_mode"] = "file"
+            doc["format"] = "mseed"
+            doc["dir"] = odir
+            doc["dfile"] = dfile
             dbh.insert_one(doc)
+
+    def save_dataframe(self, collection, df, parallel, null_values=None, one_to_one=True):
```

Comment:
To be consistent with the API of `save_data` as well as `save_textfile` below, I think we should probably change the order of parameters to `save_dataframe(self, df, collection, null_values=None, one_to_one=True, parallel=False)`